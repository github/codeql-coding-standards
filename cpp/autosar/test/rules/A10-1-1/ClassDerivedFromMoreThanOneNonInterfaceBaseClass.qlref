rules/A10-1-1/ClassDerivedFromMoreThanOneNonInterfaceBaseClass.ql