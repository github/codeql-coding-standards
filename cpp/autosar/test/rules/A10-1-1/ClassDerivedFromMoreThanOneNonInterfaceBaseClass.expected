| test.cpp:8:7:8:7 | D | Class derives from more than one non-abstract base class ($@ and $@) | test.cpp:8:11:8:11 | derivation | B | test.cpp:8:14:8:14 | derivation | A |
| test.cpp:19:7:19:7 | I | Class derives from more than one non-abstract base class ($@ and $@) | test.cpp:19:31:19:31 | derivation | B | test.cpp:19:34:19:34 | derivation | A |
